ui/startup.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+# Use envsubst to substitute environment variables in the nginx configuration
+VITE_API_TOKEN=$(cat /var/run/secrets/kubernetes.io/serviceaccount/token) envsubst < /usr/share/nginx/html/index.html > /usr/share/nginx/html/index.html.temp
+cp -f /usr/share/nginx/html/index.html.temp /usr/share/nginx/html/index.html
```

Comment:
I am partial to a `mv` here instead of a copy. This would be marginally cleaner since it eliminates the unused file on the system.

Reply:
Changed cp to mv